src/PointwiseFunctions/AnalyticData/GrMhd/CcsnCollapse.cpp

```diff
@@ -576,23 +573,34 @@ CcsnCollapse::variables(
   using std::sin;
   // Omega = omega0/(1 + (r/A)^2)
   // vphi = distance_from_rotation_axis * Omega
-  auto rotational_velocity =
+  // Temporarily store rotational velocity in z velocity component;
+  // will overwrite at the end.
+  get<2>(spatial_velocity) =
       vars->radius * vars->sin_theta * central_angular_velocity_ /
       (1.0 + square(vars->radius * inv_diff_rot_parameter_));
 
+  // speed of light check
+  if (UNLIKELY(max(get<2>(spatial_velocity)) > 1.0)) {
```

Comment:
Should this be an `ASSERT` or an `ERROR`? The reason I ask is `ERROR` is meant to deal with things like input file problems, while `ASSERT` is more meant to deal with code bugs. `ASSERT` checks disappear in release mode and increase performance, while `ERROR`s don't and so add a performance penalty. I _think_ this seems like it should be an `ASSERT` but up to you!